Core/Object Arts/Dolphin/IDE/Base/SmalltalkSearchPresenter.cls

```diff
@@ -154,47 +151,47 @@ rangeToCompleteAt: anInteger
 				ifFalse: [anInteger to: view caretPosition - 1]]!
 
 searchItemsStartingWith: aString maxItems: anInteger
-	| result classes filter |
-	aString first isLowerCase
-		ifTrue: 
-			[| matchingSymbols |
-			matchingSymbols := self selectorsStartingWith: aString maxItems: anInteger.
-			^matchingSymbols].
-	filter := (aString includes: $>)
+	| result classes classFilter addBlock |
+	aString first isUppercase ifFalse: [^self selectorsStartingWith: aString maxItems: anInteger].
```

Comment:
Is the motivation for this change to switch from looking for any lowercase letter to any non-uppercase character (which would therefore include underscore)? If so it might be worth noting that in a comment, as it is a bit obscure otherwise.